safe/impact_function/test/test_impact_function.py

```diff
@@ -113,6 +113,7 @@
 from safe.utilities.gis import wkt_to_rectangle
 from safe.utilities.utilities import readable_os_version
 from safe.impact_function.impact_function import ImpactFunction
+from safe.impact_function.impact_function_utilities import check_input_layer
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->